tree/ntuple/test/ntuple_serialize.cxx

```diff
@@ -2059,3 +2059,25 @@ TEST(RNTuple, SerializeMultiColumnRepresentationDeferredInMainHeader)
    EXPECT_EQ(expect2_0, columnRange2_0);
    EXPECT_EQ(expect2_1, columnRange2_1);
 }
+
+TEST(RNTuple, SerializeAttrSets)
```

Comment:
Perhaps add a test that serializes/deserializes an entire ntuple descriptor with attribute sets.